apps/chat/src/components/Header/Header.tsx

```diff
@@ -53,20 +65,62 @@ const Header = () => {
   );
 
   const handleToggleChatbar = useCallback(() => {
-    if (!showChatbar && isSmallScreen()) {
+    if (!showChatbar && isMediumScreen()) {
       dispatch(UIActions.setShowPromptbar(false));
+    }
+
+    if (!showChatbar && isSmallScreen()) {
       dispatch(UIActions.setIsProfileOpen(false));
     }
+
+    if (!showChatbar && !isMediumScreen()) {
```

Comment:
```suggestion
    if (!showChatbar && !isMediumScreen() && showPromptbar) {
```

Reply:
It is needed to reduce Promptbar width even if it is not visible. Othewise, it will be "jumping" if to open Promtbar and close-open Chatbar.